Tests/Shared.Specs/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -1033,13 +1036,13 @@ public void
             //-----------------------------------------------------------------------------------------------------------
             // Act
             //-----------------------------------------------------------------------------------------------------------
-            Action action = () => throwLongerThanWaitTime.Should()
+            Action action = () => throwLongerThanWaitTime.Should(clock)
                 .NotThrowAfter(waitTime, pollInterval, "we passed valid arguments");
 
             //-----------------------------------------------------------------------------------------------------------
             // Assert
             //-----------------------------------------------------------------------------------------------------------
-            action.Should().Throw<XunitException>()
+            action.Should(clock).Throw<XunitException>()
```

Comment:
🤔 `Should().Throw()` and `Should().NotThrow()` should not need an `ITimer` as they aren't timing related assertions?

Reply:
Indeed!